resources/git_credentials.rb

```diff
@@ -0,0 +1,41 @@
+resource_name :git_credentials
+
+property :path,  String, name_property: true
+property :owner, String, default: 'root'
+property :group, String, default: 'root'
+property :mode, [String, Integer], default: '0600'
+property :secrets_databag, String, default: node['osl-git']['secrets_databag']
+property :secrets_item,    String, default: node['osl-git']['secrets_item']
+
+default_action :create
+
+action :create do
+  git_config 'credential.helper' do
+    value "store --file #{new_resource.path}"
+    scope 'global'
+    user  new_resource.owner
+    group new_resource.group
+  end
```

Comment:
Sigh, this is the second time I'm writing this comment (it didn't go through the first time for some reason). Basically, there's a problem here when you set two paths for the same user since you can only have one store path set per user. Take a look at ``/root/.gitconfig`` in the attribute suite to see what I mean.

I think the real solution needs to be making ``user`` the name attribute instead of ``path``.